### PR TITLE
Fix any in TS: part 1

### PR DESCRIPTION
**Pull request by rndD**
Hi!
I wanna help konva become free of `any` type. So it's the first part of a series of PRs.

### CODE REVIEW

File: src/Tween.ts [hunk 1]
```diff
@@ -280,11 +281,11 @@ export class Tween {
         if (end.length > start.length) {
           // so in this case we will increase number of starting points
           trueStart = start;
-          start = Util._prepareArrayForTween(start, end, node.closed());
+          start = Util._prepareArrayForTween(start, end, (node as Line).closed());
```
Comment (rndD) [inline src/Tween.ts]: I'm not sure here. Is it a really logic about `line`? Mb I can do a type guard helper above.

Comment (lavrton) [reply src/Tween.ts]: Probably we can just wite `node.getAttr('closed')` here.

Comment (rndD) [reply src/Tween.ts]: Are there any test which covers it? 
Can you tell me how can I test it?

Comment (lavrton) [reply src/Tween.ts]: Related tests are here: https://github.com/konvajs/konva/blob/master/test/unit/Tween-test.js#L255

**Comment by rndD** [comment]
Do I need to fix smth else?

**Comment by lavrton** [comment]
Those are good updates. Thanks for your PR.

**Pull request merged**